onnx/test/version_converter_test.py

```diff
@@ -68,6 +70,432 @@ def test():  # type: () -> None
             self._converted(graph, helper.make_operatorsetid("", 8), 6)
         self.assertRaises(RuntimeError, test)
 
+    # Test Add Adapter: 8 -> 5
+    def test_add_7_5(self):  # type: () -> None
+        nodes = [helper.make_node('Add', ["X1", "X2"], ["Y"])]
+        graph = helper.make_graph(
+            nodes,
+            "test",
+            [helper.make_tensor_value_info("X1", TensorProto.FLOAT, (5,)),
+                helper.make_tensor_value_info("X2", TensorProto.FLOAT, (1,))],
+            [helper.make_tensor_value_info("Y", TensorProto.FLOAT, (5,))])
+        converted_model = self._converted(graph, helper.make_operatorsetid(
+            "", 8), 5)
+        # Assert equality of graph and converted_model
+        assert converted_model.graph.node[0].op_type == "Add"
+        assert converted_model.opset_import[0].version == 5
+
+    # Test Add Adapter: 5 -> 8
+    def test_add_5_7(self):  # type: () -> None
+        nodes = [helper.make_node('Add', ["X1", "X2"], ["Y"])]
+        graph = helper.make_graph(
+            nodes,
+            "test",
+            [helper.make_tensor_value_info("X1", TensorProto.FLOAT, (5,)),
+                helper.make_tensor_value_info("X2", TensorProto.FLOAT, (1,))],
+            [helper.make_tensor_value_info("Y", TensorProto.FLOAT, (5,))])
+        converted_model = self._converted(graph, helper.make_operatorsetid(
+            "", 5), 8)
+        # Assert equality of graph and converted_model
+        assert converted_model.graph.node[0].op_type == "Add"
+        assert converted_model.opset_import[0].version == 8
+
+    # Test Mul Adapter: 8 -> 6
+    def test_mul_7_6(self):  # type: () -> None
```

Comment:
We can do the same merge here, from 8->5 and 5->8 on mul and other ops which have more than one converters.